<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyDisplayGlyphWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyDisplayGlyphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="GlyphTypeLabel">
     <property name="text">
      <string>Glyph Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="GlyphTypeSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="GlyphEigenvectorLabel">
     <property name="text">
      <string>Glyph Eigenvector</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="GlyphEigenvectorSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ScaleFactorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Scale factor</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkSliderWidget" name="ScaleFactorSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="pageStep">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="SpacingLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Spacing</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkSliderWidget" name="SpacingSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="pageStep">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QStackedWidget" name="GlyphSubPropertiesWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Lines"/>
     <widget class="QWidget" name="Tubes">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="GlyphSidesLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Number of Sides</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ctkSliderWidget" name="GlyphSidesSlider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="pageStep">
          <double>1.000000000000000</double>
         </property>
         <property name="minimum">
          <double>3.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="GlyphRadiusLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ctkSliderWidget" name="GlyphRadiusSlider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Ellipsoids"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>GlyphTypeSelector</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>GlyphSubPropertiesWidget</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
